sql/derived_tables/po_lines_fund_distribution.sql

```diff
@@ -0,0 +1,45 @@
+-- This derived table shows data of the fund distribution of the purchase order lines
+
+DROP TABLE IF EXISTS po_lines_fund_distribution;
+
+CREATE TABLE po_lines_fund_distribution AS
+SELECT
+    po_lines.id AS pol_id,
+    json_extract_path_text(json_array_elements(po_lines.data->'fundDistribution'), 'fundId') AS pol_distr_fund_id,
+    json_extract_path_text(json_array_elements(po_lines.data->'fundDistribution'), 'value') AS pol_distr_value,
```

Comment:
It looks like `pol_distr_value` is a number.  If it is a monetary amount, I suggest casting to `numeric(19,4)`.  Otherwise `numeric` might be the best choice.

Reply:
Sorry, I looked at this some more.   If `pol_distr_value` is a percentage, then it should probably be `numeric(19, 2)`.

If `pol_distr_type` determines the type of value in `pol_distr_value`, then I think the `pol_distr_value` should be `numeric`.

Reply:
Thank you @nassibnassar for the feedback. The value can be an amount if you selected in the user interface amount or it can be percentage if you selected percentage. This is very differently selected by the user. 
You could do calculations and unify it. On column with the calculated amount and another one for the calculated percentage. However, this calculation brings with it many problems if you have ongoing orders. In the calculation you need the estimate price (or other totals) for 100%, but that value changes with the fiscal year rollover. The fiscal year rollover process overwritten the po lines and also the estimated price. The calculation would be wrong after the rollover. For this reason, this derived table only outputs the value and no calculations. I see no problem in changing the data type. The only question would be with what precision? numeric(19, 4) or numeric(19, 2)?

Reply:
If it will **only** be a monetary amount or a percentage, then `numeric(19, 4)` would work for both.  If it could be other kinds of numbers, then just plain `numeric`.  Thanks!